### PR TITLE
[Dataprotection] Add stop/resume protection commands and API version upgrade

### PR DESCRIPTION
**Pull request by akshayneema**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Dataprotection

File: src/dataprotection/setup.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
     description='Microsoft Azure Command-Line Tools DataProtectionClient Extension',
     author='Microsoft Corporation',
     author_email='azpycli@microsoft.com',
-    url='https://github.com/Azure/azure-cli-extensions/tree/main/src/dataprotection',
+    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/dataprotection',
```
Comment (necusjz) [inline src/dataprotection/setup.py:51]:
```suggestion
    url='https://github.com/Azure/azure-cli-extensions/tree/main/src/dataprotection',
```

**Comment by necusjz** [comment]
@akshayneema, please update HISTORY.rst as well.

File: src/dataprotection/setup.py (hunk 1 above)
Comment (akshayneema) [reply src/dataprotection/setup.py:51]: This was an autogenerated change. Is this needed to be reverted?

Comment (necusjz) [reply src/dataprotection/setup.py:51]: ya, since the current one is invalid.

**Review by necusjz — APPROVED** [review]

**Comment by kairu-ms** [comment]
@akshayneema Please add `azure-mgmt-resourcegraph` in vendor SDK, otherwise it will not work on mac and virtual environment.

**Pull request merged**